app/components/autocomplete/channel_mention/channel_mention.tsx

```diff
@@ -217,8 +217,8 @@ const ChannelMention = ({
     , [style, maxListHeight]);
 
     const runSearch = useMemo(() => debounce(async (sUrl: string, term: string) => {
+        const {channels: receivedChannels, error} = await searchChannels(sUrl, term, isSearch);
         setLoading(true);
-        const {channels: receivedChannels, error} = await searchChannels(sUrl, term);
```

Comment:
Not sure the reasoning of this change, but if you start searching before setting the loading to true, if the connection is slow, the loading spinner will take some time to appear, which is not desired.

Reply:
Fixed. I'm not sure how I did that.